docs/nodes/configuration.md

```diff
@@ -202,18 +221,34 @@ pprof-laddr = ""
 #######################################################
 [p2p]
 
+# Enable the new p2p layer.
+disable-legacy = false
+
+# Select the p2p internal queue
+queue-type = "priority"
+
 # Address to listen for incoming connections
 laddr = "tcp://0.0.0.0:26656"
 
 # Address to advertise to peers for them to dial
 # If empty, will use the same port as the laddr,
 # and will introspect on the listener or use UPnP
-# to figure out the address.
+# to figure out the address. ip and port are required
+# example: 159.89.10.97:26656
 external-address = ""
 
 # Comma separated list of seed nodes to connect to
+# We only use these if we can’t connect to peers in the addrbook
+# NOTE: not used by the new PEX reactor. Please use BootstrapPeers instead.
+# TODO: Remove once p2p refactor is complete
+# ref: https:#github.com/tendermint/tendermint/issues/5670
```

Comment:
I think seeds/bootstrap peers are probably treated similarly? Bootstrap peers are a better name for it.